org-code-javabuilder/theater/src/main/java/org/code/theater/Scene.java

```diff
@@ -0,0 +1,436 @@
+package org.code.theater;
+
+import static org.code.theater.support.DrawImageAction.UNSPECIFIED;
+
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.List;
+import org.code.media.*;
+import org.code.theater.support.*;
+
+public class Scene {
+  private static final int WIDTH = 400;
+  private static final int HEIGHT = 400;
+
+  // Visible for testing
+  static final Font DEFAULT_FONT = Font.MONO;
+  static final FontStyle DEFAULT_FONT_STYLE = FontStyle.NORMAL;
+  static final int DEFAULT_TEXT_HEIGHT = 20;
+  static final Color DEFAULT_COLOR = Color.BLACK;
+  static final double DEFAULT_STROKE_WIDTH = 1.0;
+
+  private final List<SceneAction> actions;
+
+  private Font font;
+  private FontStyle fontStyle;
+  private int textHeight;
+  private Color textColor;
+  private Color strokeColor;
+  private Color fillColor;
+  private double strokeWidth;
+
+  public Scene() {
+    this.actions = new ArrayList<>();
+
+    this.font = DEFAULT_FONT;
```

Comment:
style nit: you user `Scene.WIDTH` and `Scene.HEIGHT` below, but just `DEFAULT_FONT` here. Can we pick one pattern for static variables?